src/Instrumentation/ReactHTTP/examples/http+async_nonblocking.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+use OpenTelemetry\API\Common\Time\Clock;
+use OpenTelemetry\API\Trace\Propagation\TraceContextPropagator;
+use OpenTelemetry\Context\Context;
+use OpenTelemetry\SDK\Common\Export\Stream\StreamTransportFactory;
+use OpenTelemetry\SDK\Resource\ResourceInfoFactory;
+use OpenTelemetry\SDK\Sdk;
+use OpenTelemetry\SDK\Trace\Sampler\AlwaysOnSampler;
+use OpenTelemetry\SDK\Trace\SpanExporter\ConsoleSpanExporter;
+use OpenTelemetry\SDK\Trace\SpanProcessor\BatchSpanProcessor;
+use OpenTelemetry\SDK\Trace\TracerProvider;
+use function React\Async\async;
+use function React\Async\await;
+use React\EventLoop\Loop;
+use React\Http\Browser;
+use React\Http\Message\Request;
+
+require_once dirname(__DIR__) . '/vendor/autoload.php';
+
+$transport = (new StreamTransportFactory())->create('php://output', 'application/json');
+$exporter = new ConsoleSpanExporter($transport);
+
+$tracerProvider = new TracerProvider(
+    new BatchSpanProcessor($exporter, Clock::getDefault()),
+    new AlwaysOnSampler(),
+    ResourceInfoFactory::emptyResource(),
+);
+
+Sdk::builder()
+    ->setTracerProvider($tracerProvider)
+    ->setPropagator(TraceContextPropagator::getInstance())
+    ->setAutoShutdown(true)
+    ->buildAndRegisterGlobal();
+
+$context = Context::getCurrent();
+
+$root = $tracerProvider->getTracer('react-http-demo')->spanBuilder('root')->startSpan();
+
+Loop::futureTick(async(static function () use ($context, $root) {
+    $contextScope = $context->activate();
+    $rootScope = $root->activate();
+    try {
+        $browser = new Browser();
+
+        $requests = [
+            new Request('GET', 'https://postman-echo.com/get'),
+            new Request('POST', 'https://postman-echo.com/post'),
+            new Request('GET', 'https://httpbin.org/does-not-exist'),
+            new Request('GET', 'https://httpbin.org/get'),
+            new Request('PUT', 'localhost:2222/not-found'),
+        ];
+
+        foreach ($requests as $request) {
+            try {
+                $response = await($browser->request($request->getMethod(), $request->getUri()));
```

Comment:
This is still blocking; should the `foreach` loop body be wrapped in a `Loop::futureTick()`?

Reply:
It's non-blocking in the context of other ReactPHP EventLoop events. I'll add some additional output in the next commit to make this more obvious.

The `foreach` loop is ultimately wrapped in a `Loop::futureTick()` at:
https://github.com/open-telemetry/opentelemetry-php-contrib/blob/40248282bd448bf331dc470738ccd9c81e193f25/src/Instrumentation/ReactHTTP/examples/http%2Basync_nonblocking.php#L42

Reply:
The `http+async_blocking.php` example is also non-blocking in the context of other event loop callbacks; IMO we should drop one of the two examples if they only differ in the `Loop::futureTick(async(...))` call.

Reply:
The difference is without the `Loop::futureTick(async(...))` call, the blocking example is not _in_ the event loop. It was included to match https://reactphp.org/http/#blocking but I'm also completely okay reducing the number of examples.